mongo/mongo.go

```diff
@@ -256,10 +263,21 @@ func (m *Mongo) checkoutConnection(server driver.Server) (conn driver.Connection
 	return conn, nil
 }
 
+func (m *Mongo) unacknowledgedRoundTrip(conn driver.Connection, req []byte) (err error) {
+	addressTag := getStatsAddressTag(conn)
```

Comment:
I took a guess here about what stats this should publish. Should this also be publishing a time like:

```
m.statsd.Timing("unacknowledged_round_trip", time.Since(start)...)
```

Reply:
Thanks, looks good. I think a timing metric would be useful too... perhaps use `"round_trip"` but pass `unacknowledged:true` as a tag?

Reply:
Done.